content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -192,6 +192,7 @@ metadata:
   name: foo-pvc
   namespace: foo
 spec:
+  storageClassName: "" # Empty string must be explicitly set otherwise default StorageClass will be set
```

Comment:
Changes look good.
nit: You can add the yaml attribute to the code block to enable code highlighting.

```yaml
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: foo-pvc
  namespace: foo
spec:
  storageClassName: "" # Empty string must be explicitly set otherwise default StorageClass will be set
  volumeName: foo-pv
...
```

Reply:
done